src/templates/widgets/background_fill.html

```
@@ -0,0 +1,20 @@
+    <legend class="form-label">Background Fill</legend>
+    <div class="radio-segment">
+        <label class="radio-option">
+            <input type="radio" id="backgroundOptionBlur" name="backgroundOption" value="blur" checked>
+            <span>Blur</span>
+        </label>
+        <label class="radio-option">
+            <input type="radio" id="backgroundOptionColor" name="backgroundOption" value="color">
+            <span>Solid Color</span>
+        </label>
+        <input
+            type="color"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)